lib/src/Pos/Util/Config.hs

```diff
@@ -2,6 +2,7 @@
 
 module Pos.Util.Config
        ( embedYamlConfigCT
+       , embedYamlConfig
```

Comment:
What's the difference between `embedYamlConfigCT` and `embedYamlConfig`? Do we need both?

Reply:
I left `embededYamlConfigCT` just in case, it may be needed in future